Code/Mantid/Framework/DataHandling/src/LoadMcStas.cpp

```diff
@@ -543,24 +543,24 @@ void LoadMcStas::readHistogramData(
  */
 int LoadMcStas::confidence(Kernel::NexusDescriptor &descriptor) const {
   using namespace ::NeXus;
```

Comment:
This function returns a number indicating its confidence that the file being loaded is a McStas file.